src/Organizer/OrganizerLDProjector.php

```diff
@@ -698,17 +699,11 @@ private function updateCompleteness(JsonDocument $jsonDocument): JsonDocument
     {
         $body = $jsonDocument->getAssocBody();
 
-        $completeness = 0;
-        foreach ($this->weights as $key => $weight) {
-            if (!isset($body[$key])) {
-                continue;
-            }
-
-            if ($key === 'contactPoint' && $this->isContactPointEmpty($body[$key])) {
-                continue;
-            }
+        $completeness = $this->completeness->forDocument($jsonDocument);
 
-            $completeness += $weight;
+        if (isset($body['contactPoint']) && $this->isContactPointEmpty($body['contactPoint'])) {
```

Comment:
Ipv hier een vast edge case te programmeren, kunnen we dit niet mee in de weights config steken?
En dan bv als logica zeggen, als de weight negatief is, dan krijg je die minus punten als het veld leeg is.
Of, alternatief, een tweede weights config geven.

Dan hebben we een veel flexibeler systeem voor de toekomst.

Reply:
After doing an extra check in the code I noticed this also happens for places and events, so I added this logic inside the calculate method.